client/src/components/ChannelInfo/index.js

```diff
@@ -0,0 +1,165 @@
+import React, { useState, useEffect } from "react";
+import PropTypes from "prop-types";
+import { Tabs, Tab, Typography, Box } from "@material-ui/core";
+import Cookies from "js-cookie";
+import axios from "axios";
+import { githubApiDomain } from "../../utils/constants";
+
+import "./index.css";
+
+function TabPanel(props) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`channelinfo-tabpanel-${index}`}
+      aria-labelledby={`channelinfo-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box style={{ padding: "15px 20px 0px 20px" }}>
+          <Typography component={"span"}>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  );
+}
+
+TabPanel.propTypes = {
+  children: PropTypes.node,
+  index: PropTypes.any.isRequired,
+  value: PropTypes.any.isRequired,
+};
+
+function a11yProps(index) {
+  return {
+    id: `simple-tab-${index}`,
+    "aria-controls": `simple-tabpanel-${index}`,
+  };
+}
+
+export default function ChannelInfo(props) {
+  const [repoInfo, setRepoInfo] = useState({});
+  const [isPrivate, setIsPrivate] = useState(false);
+
+  useEffect(() => {
+    const ghRepoInfo = async () => {
+      try {
+        // Fetches repository information
+        const repository = props.location.pathname
+          .split("/")[2]
+          .replace("_", "/");
+        const headers = {
+          accept: "application/json",
+        };
+        if (Cookies.get("gh_private_repo_token")) {
+          headers["Authorization"] = `token ${Cookies.get(
+            "gh_private_repo_token"
+          )}`;
+        }
+        const ghRepoInfoResponse = await axios({
+          method: "get",
+          url: `${githubApiDomain}/repos/${repository}`,
+          headers: headers,
+        });
+        setRepoInfo(ghRepoInfoResponse.data);
+      } catch (error) {
+        console.log(error);
+        setIsPrivate(true);
+      }
+    };
+    ghRepoInfo();
+  }, [props.location.pathname]);
+
+  const [value, setValue] = useState(0);
+  const [activeTab, setActiveTab] = useState(0);
+  const repoURL = `https://github.com/${props.location.pathname
+    .split("/")[2]
+    .replace("_", "/")}`;
+
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+    setActiveTab(newValue);
+  };
+
+  return (
+    <div>
```

Comment:
Give this div a  max-height to 50% of the right sidebar.
Don't forget to test the div with repo description of height greater than the max-height of div and handle the overflow

Reply:
# When no overflow:

![Screenshot from 2021-03-08 00-30-44](https://user-images.githubusercontent.com/28918901/110251211-915c2a80-7fa5-11eb-87b4-83f1c3ebd7d1.png)


# When overflow: 
![Screenshot from 2021-03-08 00-27-47](https://user-images.githubusercontent.com/28918901/110251144-2b6fa300-7fa5-11eb-9a46-4b96134d3718.png)